### ISSUE TITLE
Migrate dbt docs deployment to GitHub

### ISSUE THREAD

**Issue by pnadolny13** [meltano/squared#253]
As part of https://github.com/meltano/squared/issues/250 and https://github.com/meltano/squared/pull/252 all CI workflows were migrated to GitHub actions but dbt docs deployment to gitlab pages was skipped since its not critical for the first iteration.

- Should we move to GitHub pages?
- Should we move to Netlify? Since we started moving other things like the hub over there

cc @tayloramurphy

**Comment by tayloramurphy** [comment]
@pnadolny13 we have https://github.com/meltano/internal-general/issues/247 which is about moving to Cloudflare. I'll need input on @aaronsteers before we can really move forward.

**Comment by tayloramurphy** [comment]
@pnadolny13 we have https://github.com/meltano/internal-general/issues/247 which is about moving to Cloudflare. I'll need input from @aaronsteers before we can really move forward.

**Comment by tayloramurphy** [comment]
Bumped to August for priority. We should be adding documentation in the markdown and YAML for dbt docs, but getting the docs deployed is slightly lower priority. This could come up during Managed Meltano too since that's theoretically something users would want to deploy!

**Comment by aaronsteers** [comment]
A GitHub Pages implementation would be fantastic:

https://meltano.slack.com/archives/CFG3C3D1Q/p1659979001724969?thread_ts=1659907095.791609&cid=CFG3C3D1Q
